### PR TITLE
Rw results bugs testing

### LINKED ISSUE #44: Add proptypes to Results

### LINKED ISSUE #46: add Proptypes to drinkCards

### LINKED ISSUE #53: Clear recipe data when returning to results screen

### LINKED ISSUE #54: Create a message on page load before search

### LINKED ISSUE #58: Redo testing for Results

### LINKED ISSUE #61: Redo Card Testing

### PR DESCRIPTION
**Pull request by rwilliams659**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
- Fixes Results component display for drinksList to only have label for non-alcoholic drinks 
- Adds propTypes for Results component
- Adds propTypes for DrinkCard component
- Completes testing for Results component
- Completes testing for DrinkCard component
- Adds Welcome message to Results on page load
- Adds drinkRecipe prop clearing on the Route for homepage 

### Files modified:
- Action creators
- App component
- Drink Card css
- Drink card component
- Drink card test
- Results css 
- Results component
- Results test
- drinksRecipe & drinksList reducers

### Has This Been Tested?
- [ ] No
- [x] Yes

### Are there specific methods for testing?
Run npm test 

### What are the relevant tickets?
resolve #44 
resolve #46 
resolve #53 
resolve #54 
resolve #58 
resolve #61 

### Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -29,12 +29,15 @@ class App extends Component {
...
-          )}
+          render={() => {
+            this.props.resetRecipe(); 
+            return (
+              <>
+                <Search />
+                <Results />
+              </>
+            )
+          }}
```
Comment (relyt4me) [inline src/components/App/App.js:40]: Glad this worked easily

**Review by relyt4me — APPROVED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (tarynmartin) [reply src/components/App/App.js:40]: This is such a great way to clear the recipe history!

File: src/components/DrinkCard/DrinkCard.css [hunk 2]
```diff
@@ -22,15 +22,16 @@
...
   background-color: #c3f584;
-}
+} */
 
-.non-alcoholic {
-  background-color: #ffd271;
-}
+/* .non-alcoholic {
+  
+} */
```
Comment (tarynmartin) [inline src/components/DrinkCard/DrinkCard.css:34]: could probably take out the above if not using at all

**Review by tarynmartin — APPROVED** [review]
Looks great, good job with all that testing!

**Pull request merged**